message_sender/views.py

```diff
@@ -182,12 +197,26 @@ def post(self, request, *args, **kwargs):
             message.delivered = True
             message.metadata["delivery_timestamp"] = request.data["timestamp"]
             message.save(update_fields=['delivered', 'metadata'])
+
+            # OBD number of successful tries metric
+            if "voice_speech_url" in message.metadata:
+                fire_metric.apply_async(kwargs={
+                    "metric_name": 'vumimessage.obd.successful.sum',
+                    "metric_value": 1.0
+                })
```

Comment:
The same here for Junebug, it's called a better name in Junebug, `submitted` and `rejected`.